component/DateSelectionMonthlyCalendar.js

```diff
@@ -51,10 +51,21 @@ export default () => {
 
   const nextMonth = () => {
     dispatch('selectedDate/set', new Date(year, month + 1, 1));
+    dispatch('initialNextFreeSlotsLoading/set', true);
   }
 
   const previusMonth = () => {
-    dispatch('selectedDate/set', new Date(year, month, 0));
+    dispatch('selectedDate/set', new Date(year, month - 1, 1));
```

Comment:
Ezt szerintem tegyük a for után. (tehát 1. megkeressük az első napot ahol van next free slot és oda megyünk 2. ha nem volt ilyen akkor elseje)